pkg/reader/proc/proc.go

```diff
@@ -277,11 +277,24 @@ func LogCurrentSecurityContext() {
 		}
 	}
 
+	lockdown := ""
+	data, err := os.ReadFile("/sys/kernel/security/lockdown")
+	if err == nil && len(data) > 0 {
+		values := strings.TrimSpace(string(data))
+		i := strings.Index(values, "[")
+		j := strings.Index(values, "]")
+		if i >= 0 && j > i {
+			lockdown = values[i+1 : j]
+			logLSM = true
+		}
+	}
```

Comment:
How about emitting a warning if lockdown is == "confidentiality"?

Maybe something like
"lockdown mode seems to be set to 'confidentiality', which does not allow Tetragon to operate because it prohibits loading BPF programs. See https://github.com/iovisor/bcc/issues/2565 for more details".

Reply:
Ok make sense, I added it thank you!

Also created a doc issue for it https://github.com/cilium/tetragon/issues/2265